.github/workflows/test_precommit.yml

```diff
@@ -148,7 +148,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04, macos-12]
-        python-version: [3.8, 3.9, '3.10', '3.11']
+        python-version: [3.9, '3.10', '3.11']
```

Comment:
Can 3.8 be kept here and in `model_api/python/pyproject.toml`? It's not costly to keep the support

Reply:
Yes, that's possible, but my intention was to ease type annotations in new PRs.
However, OV 2024.1 still supports 3.8, probably they'll wait till October, and then drop it. Let's keep 3.8 until that point then

Reply:
Any feature would be a stronger reason to drop 3.8 compared to just keeping the support. You are the only users of model_api. So if you don't need 3.8, feel free to drop now.